src/main/java/uk/gov/hmcts/ccd/data/definition/HttpUIDefinitionGateway.java

```diff
@@ -182,6 +183,30 @@ public SearchResult getWorkBasketResult(int version, String caseTypeId) {
         }
     }
 
+    @Override
+    public SearchResult getSearchCasesResult(int version, String caseTypeId, UseCase useCase) {
+        try {
+            final Instant start = Instant.now();
```

Comment:
have you got `final` configured in your IDE? I think `final` added everywhere really makes the code heavy to read. I think it's not worth it